vscode/package.json

```diff
@@ -26,6 +28,21 @@
   ],
   "main": "./out/extension.js",
   "contributes": {
+    "chatParticipants": [
+      {
+        "id": "rubyLsp.chatAgent",
+        "fullName": "Ruby",
+        "name": "ruby",
+        "description": "How can I help with your Ruby application?",
```

Comment:
Hello! 👋 Out of curiosity, would it be better to specify "Ruby on Rails application"? Scanning the PR, the prompt seems to be more Rails specific applications rather than plain Ruby applications.